src/main/java/org/mskcc/limsrest/service/requesttracker/ProjectSampleTree.java

```diff
@@ -48,8 +48,8 @@ public WorkflowSample getRoot() {
         return root;
     }
 
-    public Boolean isPassedDataQc() {
-        return SEQ_QC_STATUS_PASSED.equalsIgnoreCase(this.dataQcStatus);
+    public Boolean isQcIgoComplete() {
```

Comment:
maybe boolean instead of Boolean since looks like it should never be null

Reply:
Thanks will do